README.md

```diff
@@ -8,7 +8,7 @@
 <br />
 <p align="center">
   <a href="https://github.com/Builditluc/wiki-tui">
-    <img src= "https://raw.githubusercontent.com/Builditluc/wiki-tui/main/logo.png" alt="Logo" width="234" height="167">
+    <img src="https://raw.githubusercontent.com/Builditluc/wiki-tui/main/icons/hicolor/scalable/apps/wiki-tui.svg" width="50%" alt="Logo; wiki-tui entered into a command line prompt"/>
```

Comment:
For me the SVG isn't showing up in the `README`. After looking around the web for a bit I found out that many people are having problems with SVG rendering in `README` files. Could you maybe change the logo on the `README` to a `.png`?

Reply:
Ah, didn't notice that! Thank you